src/js/view/controls/controlbar.js

```diff
@@ -343,6 +346,7 @@ export default class Controlbar {
             this.elements.volumetooltip.volumeSlider.render(muted ? 0 : vol);
             utils.toggleClass(this.elements.volumetooltip.element(), 'jw-off', muted);
             utils.toggleClass(this.elements.volumetooltip.element(), 'jw-full', vol >= 75 && !muted);
+            this.elements.volumetooltip.element().setAttribute('aria-label', `volume ${muted ? 0 : vol}%`);
```

Comment:
If you keep accessing the same property more than three times and these are lengthy strings that won't minify, it's a good rule of thumb to put them in a var, and replace all instances of them in the function:
```js
const mute = this.elements.mute;
const volumetooltip = this.elements.volumetooltip;
if (mute) {
utils.toggleClass(mute, 'jw-off', muted);
...
if (volumetooltip) {
  volumeSlider.render(muted ? 0 : vol);
...